### DISCUSSION TITLE
Can I withdraw my transaction from the wrong address?

### DISCUSSION THREAD

**Discussion by Thzen1105** [Chia-Network/chia-blockchain#8937] [Support]
Hi , I accidentally sent my xch to an address that i do not own , can i withdraw my xch ?
![chia](https://user-images.githubusercontent.com/93151031/138768747-e10410f5-0a1f-4a99-876f-6f7c80159e43.png)
![chia2](https://user-images.githubusercontent.com/93151031/138768757-36bb6a36-0bcc-4d22-957b-3e9aa8ea3d0e.png)

**Comment by shaneo257** [comment]
sadly no
